client/db.go

```diff
@@ -419,6 +419,19 @@ func (db *DB) Txn(retryable func(txn *Txn) error) error {
 	return newTxn(*db, 1 /* depth */).exec(retryable)
 }
 
+// BeginTxn starts a new transaction.
+func (db *DB) BeginTxn() *Txn {
+	return newTxn(*db, 1 /* depth */)
```

Comment:
Not sure what you meant. Added three methods NewTxn, NewTxnFromProto, and Txn.ToProto.